src/cli/setupServer.ts

```diff
@@ -250,14 +232,42 @@ export const setupServer = async ({
     })
 
     if (config.refillingWallets) {
```

Comment:
Can we leave a comment in the config that this must only be enabled on a single instance. Otherwise all the instance will try to send money to all the wallets.